Projects/Crypto_Price_Tracker/README.md

```diff
@@ -0,0 +1,5 @@
+npm install
+npm start 
```

Comment:
Why this two lines are there in the README.md??
Is these needed??

Reply:
> Why this two lines are there in the README.md?? Is these needed??

It is needed if the user wants to run the project on their device

Reply:
Should I make a more elaborate readme.md with an instruction on how to clone and all for this project?

Reply:
No . Don't make a very elaborate README file just make it to the point.